### PR TITLE
cast the wei budget to int

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: apps/wasm/task.py [hunk 1]
```diff
@@ -583,7 +583,7 @@ def build_full_definition(
             logger.warning("Assigning task default budget: %d",
                            task_def.budget / denoms.ether)
         else:
-            task_def.budget = dictionary.get('budget') * denoms.ether
+            task_def.budget = int(dictionary.get('budget') * denoms.ether)
```
Comment (kubkon) [inline apps/wasm/task.py]: Could we do some rounding here though? It kinda hurts my eyes when I see simple truncation...

File: tests/apps/wasm/test_task.py [hunk 2]
```diff
@@ -108,6 +111,7 @@ def test_build_full_definition(self):
             WasmTaskTypeInfo(), TEST_TASK_DEFINITION_DICT,
         )
         self.assertEqual(task_def.subtasks_count, 2)
+        self.assertEqual(task_def.budget, int(0.5 * denoms.ether))
```
Comment (kubkon) [inline tests/apps/wasm/test_task.py]: Ditto!

File: apps/wasm/task.py (hunk 1 above)
Comment (shadeofblue) [reply apps/wasm/task.py]: sure... though, do we imagine specifying values to 18th decimal place in the task definition JSON? ...

**Review by mbenke — APPROVED** [review]
Seems to work with https://github.com/golemfactory/g-flite/pull/37

**Review by kubkon — APPROVED** [review]
LGTM!

**Pull request merged**